app/models/user.rb

```diff
@@ -141,8 +141,8 @@ def self.roster_create(email, first_name, last_name, school, major, year)
     user.password_confirmation = temp_pass
     user.skip_confirmation!
 
-    Rails.logger.debug("user email: ", user.email)
-    Rails.logger.debug("user pswd: ", user.password)
+    # Rails.logger.debug("user email: ", user.email)
```

Comment:
This was there before I edited it. Could you see if there's a way to log this without throwing the error?